test/test_ops.py

```diff
@@ -459,12 +459,19 @@ def test_jit_alias_remapping(self, device, dtype, op):
         # - args_kw is the string of args/kwargs used to call the op, same as args_annot_kw but
         # without type annotations
         args = ["t0"]
+
+        def parse_value(v):
```

Comment:
nit: the name is well-chosen, I think.

Reply:
You mean _not well-chosen_, correct? Do you have a better proposal? Maybe `process_value`?

Reply:
Yes, not well-chosen. The methods adapts string values, something related with that I think should be in the name.